components/sections/Pricing.tsx

```diff
@@ -81,15 +81,6 @@ const PRICING_ROWS: PricingTableRow[] = [
 			team: 'Custom',
 		},
 	},
-	{
-		desc: 'Add-on: Dedicated Database',
-		desc2: '(AWS RDS)',
-		values: {
-			community: false,
-			pro: 'Custom',
-			team: 'Custom',
-		},
-	},
```

Comment:
Rather than removing this, should it remain, but the values updated to `add-on` for all tiers? It feels like the fact that an RDS addon is available is a bit hidden on the pricing page following these changes.

Reply:
This makes sense :+1: